src/psyclone/tests/psyad/tl2ad_harness_test.py

```diff
@@ -0,0 +1,486 @@
+# -----------------------------------------------------------------------------
```

Comment:
I prefer a 1-to-1 correlation between Python files and Python test files, or at least all tests for a particular function/class in a Python file together. Here, we need to look in two files to cover a particular class/method.

Reply:
OK, I've put the contents of this file into tl2ad_test.py.